executor/executor.go

```diff
@@ -289,11 +267,11 @@ func Run(path string, env []string, emitter screwdriver.Emitter, build screwdriv
 			"EXITCODE=$?; " +
 			exportEnvCmd +
 			"echo $SD_STEP_ID $EXITCODE; }", //mv newfile to file
-		"trap finish ABRT EXIT;\necho ;\n",
+		"trap finish ABRT EXIT;echo " + guid + " $?\n",
```

Comment:
change echo command in [this](https://github.com/screwdriver-cd/launcher/blob/09a34bbc798203a552fc89cd2e67e21c94a83259/executor/executor.go#L73) format